libbeat/logp/core.go

```diff
@@ -62,6 +64,12 @@ type coreLogger struct {
 
 // Configure configures the logp package.
 func Configure(cfg Config) error {
+	return ConfigureWithOutputs(cfg)
+}
+
+// Configure configures the logp package also sets up the following outputs to also be used
+// by the logger.
+func ConfigureWithOutputs(cfg Config, outputs ...zapcore.Core) error {
```

Comment:
Let's add a note here that agent expects `ConfigureWithOutputs` to still create the output. When cleaning up/refactoring this function in the method I would move the output init to the `Configure` method, which can break logging in the agent from working correctly.

e.g.

```
func Configure(cfg Config) error {
  logOutput, err := CreateLogOutput(cfg.X)
  if err != nil { ... }
  return ConfigureWithOutputs(cfg, logOutput)
}
```